utils/pre-swap-checker/contracts/Imports.sol

```diff
@@ -8,5 +8,7 @@ import "@airswap/transfers/contracts/TransferHandlerRegistry.sol";
 import "@airswap/transfers/contracts/handlers/ERC20TransferHandler.sol";
```

Comment:
Can remove `import "@airswap/transfers/contracts/TransferHandlerRegistry.sol";` from Imports.sol - it's imported in the PreSwapChecker already.